Packs/CommonScripts/Scripts/GetIndicatorDBotScore/GetIndicatorDBotScore.py

```diff
@@ -30,16 +30,26 @@
 for entry in data:
     indicator_type = entry["indicator_type"]
     score = entry["score"]
-    source = entry.get('source', '')
+    sources = entry.get('sourceBrands', [])
     if hasattr(INDICATOR_TYPE_DICTIONARY, indicator_type):
         indicator_type = INDICATOR_TYPE_DICTIONARY[indicator_type]
 
-    ec["DBotScore"].append({
-        "Indicator": indicator,
-        "Type": indicator_type,
-        "Vendor": source,
-        "Score": score
-    })
+    if sources:
+        for source in sources:
+            ec["DBotScore"].append({
+                "Indicator": indicator,
+                "Type": indicator_type,
+                "Vendor": source,
+                "Score": score
+            })
+    else:
+        ec["DBotScore"].append({
+            "Indicator": indicator,
+            "Type": indicator_type,
+            "Vendor": '',
+            "Score": score
+        })
```

Comment:
@anara123 , the new script is good, but still, seems that when there are not sources, we still add empty DBotScore - why? Still needed?

Reply:
Yes, the indicator can have no sources and still exists in indicators page. We want to return that indicator too.

Reply:
and we have to return it on the DBotScore?
It could also be wrong, DBot didn't set it, it could have been set manually.


If you think it could be changed in content let me know. Otherwise we would add the extra prop.

Reply:
Either way we need to add new prop, plan to implement an integration that will query XSOAR indicators database and return DBotScore as any other threat intel integration.